Modules/ThirdParty/MINC/src/libminc/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ INCLUDE(CTest)
 
 ENABLE_TESTING()
 
-CMAKE_MINIMUM_REQUIRED(VERSION 3.6)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.10)
```

Comment:
This diverges from minc upstream anyway,   Why not just go with 3.16.3?

I'm OK with 3.10, but curious of the choice.

Reply:
This is the version upstream set it too. But it looks like the line moved too:
https://github.com/BIC-MNI/libminc/commit/6840ef7c579fa6d5f802874b7f0b2f965f09ac61

So it will conflict :(